execute_pbench

```diff
@@ -14,6 +14,7 @@ spacing=0
 sys_type=""
 test_name=""
 times_to_run=1
+results_dir=""
```

Comment:
This should really be pbench_results_dir since it's _only_ taking effect for pbench tests.